config/flask_config.py

```diff
@@ -3,57 +3,91 @@
-    SECRET_KEY = environ['SECRET_KEY']
-    DEBUG = True if environ['DEBUG'] == 'TRUE' else False
-
-    # credentials that allow the app to modify calendars without using a
-    # without a user's login
-    INSTALLED_APP_SECRET_PATH = environ.get('INSTALLED_APP_CLIENT_SECRET_PATH',
-                                               'client_secrets.json')
-    # where the installed credentials are stored
-    INSTALLED_APP_CREDENTIALS_PATH = environ.get('INSTALLED_APP_CREDENTIALS_PATH',
-                                   'config/credentials.json')
+    config['PORT'] = int(config['PORT'])
 
     # Google Auth
     # This is used for the webapp that allows Google+ login
-    GOOGLE_AUTH_ENABLED = True if environ['GOOGLE_AUTH_ENABLED'] == 'TRUE' \
-                            else False
-    CLIENT_SECRETS_PATH = environ.get('GOOGLE_AUTH_SECRETS',
-                                      'config/client_secrets.json')
+    config['GOOGLE_AUTH_ENABLED'] = True if config['GOOGLE_AUTH_ENABLED'] == 'TRUE' \
+                                    else False
+    GOOGLE_AUTH_ENABLED = config['GOOGLE_AUTH_ENABLED'] 
+    
     # Setup Google Auth
-    if GOOGLE_AUTH_ENABLED:
+    if config['GOOGLE_AUTH_ENABLED']:
         try:
-            with open(CLIENT_SECRETS_PATH, 'r') as f:
+            with open(config['CLIENT_SECRETS_PATH'], 'r') as f:
                 _secrets_data = json.loads(f.read())['web']
-                GOOGLE_CLIENT_ID = _secrets_data['client_id']
+                config['GOOGLE_CLIENT_ID'] = _secrets_data['client_id']
                 if not _secrets_data.get('client_secret', None):
                     print ('Google Auth config file, %s,'
-                           ' missing client secret', CLIENT_SECRETS_PATH)
+                           ' missing client secret', config['CLIENT_SECRETS_PATH'])
                     exit(1)
 
         except IOError:
-            print ("The Google client_secrets file was not found at"
-                   "'{}', check that it exists.".format(CLIENT_SECRETS_PATH))
+            print ("The Google client_secrets file was not found at '{}', "
+                   "check that it exists.".format(config['CLIENT_SECRETS_PATH']))
             exit(1)
 
     # Cross-site request forgery settings
-    CSRF_ENABLED = True if environ['CSRF_ENABLED'] == 'TRUE' else False
-    CSRF_SESSION_KEY = environ['CSRF_SESSION_KEY']
-
-    # Google Calendar credentials
-    PRIVATE_CALENDAR_ID = environ['PRIVATE_CALENDAR_ID']
-    PUBLIC_CALENDAR_ID =  environ['PUBLIC_CALENDAR_ID']
+    config['CSRF_ENABLED'] = True if config['CSRF_ENABLED'] == 'TRUE' else False
 
     # Mongo configs
-    MONGODB_SETTINGS = {'DB': environ.get('MONGO_DATABASE', 'eventum')}
+    config['MONGODB_SETTINGS'] = {'DB': config['MONGO_DATABASE']}
 
-    # Logging settings
-    LOG_FILE_MAX_SIZE = environ.get("LOG_FILE_MAX_SIZE")
-    APP_LOG_NAME = environ.get("APP_LOG_NAME")
-    WERKZEUG_LOG_NAME = environ.get("WERKZEUG_LOG_NAME")
 
 except KeyError:
```

Comment:
I think that our code won't ever throw `KeyError`s, am I right? If we set them all to initially `None`, we'll just have a bunch of `None` values in the dictionary.